cmd/crates/soroban-test/tests/fixtures/test-wasms/custom_account/Cargo.toml

```diff
@@ -12,7 +12,7 @@ crate-type = ["cdylib"]
 doctest = false
 
 [dependencies]
-soroban-sdk = { workspace = true }
+soroban-sdk = { version = "=21.7.2" }
```

Comment:
fyi @fnando @willemneal @Ifropc when we get a v22 rpc setup in tests, then we can switch this back to the workspace version which is currently v22. This is just so that contracts being deployed will deploy to the v21 core successfully.

In the future we should break test-wasms into their own workspace so it's easy to set them to a different sdk version when needed.